### PR TITLE
Set member profile to inactive

### PR DESCRIPTION
**Pull request by kayceeDev**
Signed-off-by: Ekene Nwobodo <nwobodoe71@gmail.com>

**Description**
Sets a a community member status to inactive

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Resources/Resources-grid/DataWrapper.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const DataWrapper = (WrappedComponent) => {
         }
       `
     );
-
+    console.log(props);
```
Comment (Chadha93) [inline src/sections/Resources/Resources-grid/DataWrapper.js:43]: Remove this console

File: src/sections/Resources/Resources-grid/filters.js [hunk 2]
```diff
@@ -67,7 +67,6 @@ const Navigation = (props) => {
   const technologies = counting.technology.group;
   const meshes = counting.mesh.group;
 
-
```
Comment (Chadha93) [inline src/sections/Resources/Resources-grid/filters.js:70]: Revert this file change, space...

**Review by Chadha93 — CHANGES_REQUESTED** [review]

File: src/sections/Resources/Resources-grid/filters.js (hunk 2 above)
Comment (kayceeDev) [reply src/sections/Resources/Resources-grid/filters.js:70]: This is not visible on my machine.

Comment (kayceeDev) [reply src/sections/Resources/Resources-grid/filters.js:70]: This had been sorted too. The code is well-formatted with a single space instead of a double.

**Comment by l5io** [comment]
🚀 Preview for commit 77b5ac32d9ccc3e923be6b65b54f7136e2ff5382 at: https://62ab408031bdc3208f433369--layer5.netlify.app

File: src/sections/Resources/Resources-grid/filters.js (hunk 2 above)
Comment (kayceeDev) [reply src/sections/Resources/Resources-grid/filters.js:70]: > The code automatically gets formatted when you make a commit by a pre-commit linter. You just need to make changes in the first file not other

This is adjusted now.

**Comment by l5io** [comment]
🚀 Preview for commit 73ad9f0574c8e2f1e339607f798cc5b2383ec00a at: https://62ab477c6f9d4c278e521cf9--layer5.netlify.app

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Review by debo19 — APPROVED** [review]

**Pull request merged**